### PR TITLE
Different types of images made

### PR DESCRIPTION
**Pull request by Asypher00**
Made a simple round image and an image responsive to different screen size
Screenshot (7)
s

### CODE REVIEW

File: images.html [hunk 1]
```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="/images.css">
+    <title>Document</title>
```
Comment (Sameony) [inline images.html]: Add a fitting title that describes the page

File: images.html [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+</head>
+
+<body>
+<h1>Round Image</h1>
+    <img class="img-container" src="/images/download.jpg" alt="">
+
+    <h1>Responsive image</h1>
+    <img class="img-container img-responsive" src="/images/download.jpg" alt="">
```
Comment (Sameony) [inline images.html]: provide alt texts for the img tag

**Review by Sameony — COMMENTED** [review]
suggesting minor adjustments

File: images.css [hunk 3]
```diff
@@ -0,0 +1,31 @@
+body {
+    text-align: center;
+}
```
Comment (shuntzhere) [inline images.css]: You can create a base class for this in your utility css file. This way you do not have to use text-align individually on components.

File: images.css [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+
+@media screen and (min-width : 1500px) {
+    .img-responsive{border-color: rgb(115, 243, 115);
+        width : 40rem;
+        height: 40rem;
+
+      
+  }
+    
+}
```
Comment (shuntzhere) [inline images.css:29]: Remove the extra spacings.

**Review by shuntzhere — COMMENTED** [review]
Suggested some minor improvements to the pushed code.

File: images.html (hunk 1 above)
Comment (Asypher00) [reply images.html]: done! thanks for the review!

File: images.html (hunk 2 above)
Comment (Asypher00) [reply images.html]: done! thanks for the review!

**Pull request merged**